web-console/src/components/show-json/show-json.tsx

```diff
@@ -41,7 +41,13 @@ export const ShowJson = React.memo(function ShowJson(props: ShowJsonProps) {
       const resp = await Api.instance.get(endpoint);
       let data = resp.data;
       if (transform) data = transform(data);
-      return typeof data === 'string' ? data : JSON.stringify(data, undefined, 2);
+      return typeof data === 'string'
+        ? data
+        : JSON.stringify(
+            data,
+            (_, value) => (typeof value === 'bigint' ? value.toString() : value),
```

Comment:
This is cool but it would be better to use the stringify function that comes with `json-bigint`

Just do `import * as JSONBig from 'json-bigint-native';`

and then replace all `JSON.stringify` with `JSONBig.stringify`